### ISSUE TITLE
Question: Is this repo abandoned?

### ISSUE THREAD

**Issue by andeston** [Hummer12007/brightnessctl#124]
It seems that @Hummer12007 has been gone for a while. There are multiple issues open (#90, #112, #113) that are requesting a new release.               

@PowerMan2206 pointed out that this could be due to the conflict in Ukraine. https://github.com/Hummer12007/brightnessctl/issues/113#issuecomment-2440040950

> Hummer's profile says he's from Kyiv, I don't think "releasing an update to a program so that people can see their screen brightness better" is in his priority list right about now.

I'm assuming @Hummer12007 is the only person with access to the repository. If nobody can accept PRs and @Hummer12007 doesn't come back, I have a branch [here](https://github.com/andeston/brightnessctl/tree/develop) with my PRs merged. It fixes a few bugs that have been introduced since the last release.

**Comment by Hummer12007** [comment]
Nah, wrong guess. I've been going through some stuff and will try to take a look through things this week.
